central/vulnmgmt/vulnerabilityrequest/manager/requestmgr/manager_impl.go

```diff
@@ -71,9 +72,10 @@ type managerImpl struct {
 	revertTimedDeferralsTickerDuration      time.Duration
 	revertFixableCVEDeferralsTickerDuration time.Duration
 
-	stopper    concurrency.Stopper
-	upsertSem  *semaphore.Weighted
-	seqNumLock sync.Mutex
+	stopper      concurrency.Stopper
+	upsertSem    *semaphore.Weighted
+	seqNumLock   sync.Mutex
+	ImageCVEView views.CveView
```

Comment:
Just confirming, does this need exposed outside the package or could it be `imageCVEView`?